var/spack/repos/spack_repo/builtin/packages/py_nvidia_ml_py/package.py

```diff
@@ -12,8 +12,27 @@ class PyNvidiaMlPy(PythonPackage):
     homepage = "https://www.nvidia.com/"
     pypi = "nvidia-ml-py/nvidia-ml-py-11.450.51.tar.gz"
 
-    license("Unlicense")
-
+    license("BSD-2-Clause")
+
+    version("12.575.51", sha256="6490e93fea99eb4e966327ae18c6eec6256194c921f23459c8767aee28c54581")
+    version("12.570.86", sha256="0508d4a0c7b6d015cf574530b95a62ed4fc89da3b8b47e1aefe6777db170ec8b")
```

Comment:
I cannot confirm the sha256s for these versions (e.g., using `spack checksum`).  The archive files for those versions use underscores between `nvidia`, `ml`, and `py`, not dashes so Spack's substitution won't pick them up.

With a little work -- and the implementation below of `url_for_version` (https://spack.readthedocs.io/en/latest/packaging_guide.html#version-urls) -- I can confirm the checksums.